easybuild/easyconfigs/s/snpEff/snpEff-5.0-Java-13.eb

```diff
@@ -16,6 +16,10 @@ checksums = ['2b177440eb4a2c3bbfc8f50d4adf9e405f3dcfc05a31a55c96f773f07832af34']
 
 dependencies = [('Java', '13')]
 
+# we need this to appear as an executable (i.e. in snippy easyconfig)
+postinstallcmds = ['ln -s %(installdir)s/%(name)s.jar %(installdir)s/%(name)s && chmod +x %(installdir)s/%(name)s']
+modextrapaths = {'PATH': ['']}
```

Comment:
Isn't the `snpEff` that's required by `Snippy` the script in https://github.com/tseemann/snippy/blob/master/binaries/noarch/snpEff?

You can't run a JAR file directly, so making it executable and stripping the `.jar` doesn't make much sense...

Reply:
looks like you're right, I'll try to add that executable to PATH for snippy then

Reply:
@deniskristak So we should change this PR to only bump the HTSlib dep in VCFtools 0.1.16?

Reply:
correct, snippy works with that `binaries/noarch/snpEff` you mentioned...
removing my change then.

Reply:
@boegel on the second thought, isn't it a bit against what we want? :D using attached prebuilt binaries

Reply:
ive removed my change, so it's only HTSlib now

Reply:
@deniskristak Well, it depends... I think we can still add `snpEff` as a dependency to `Snippy`, but keep using the `snpEff` script that comes with `Snippy` (but this discussion belongs in #13475)